doc/src/main/asciidoc/inc/goals/build/_fabric8-helm.adoc

```diff
@@ -59,6 +59,10 @@ This configuration section know the following subelements in order to configure
 | *engine*
 | The template engine to use
 |
```

Comment:
Could you add `fabric8.helm.chartArtifactType` also to asciidoc, please?

Reply:
I'm sorry, but could you please provide a path to file? I cannot find it at all :(

Reply:
I'm talking about the variable you've added in this PR